### PR TITLE
Integ sp7 churro fans

### PR DESCRIPTION
**Pull request by aasboddu**

### CODE REVIEW

**Review by supven01 — COMMENTED** [review]
Please explain what "Churro" refers to here.
something like (Churro is a board that hosts pwm based fan controllers)?
No comments regarding the code itself. It looks good.

**Comment by aasboddu** [comment]
> Please explain what "Churro" refers to here. something like (Churro is a board that hosts pwm based fan controllers)? No comments regarding the code itself. It looks good.

@supven01 Yes churro is a board that has fan controllers. This change is to access them and control the fans on that board.

**Review by supven01 — COMMENTED** [review]
Can you describe Churro in commit message body.

**Comment by supven01** [comment]
> > Please explain what "Churro" refers to here. something like (Churro is a board that hosts pwm based fan controllers)? No comments regarding the code itself. It looks good.
> 
> @supven01 Yes churro is a board that has fan controllers. This change is to access them and control the fans on that board.

Can you describe Churro in the commit message body?

**Comment by aasboddu** [comment]
@supven01 modified the commit body. Kindly review it.

**Review by supven01 — APPROVED** [review]

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts [hunk 1]
```diff
@@ -539,3 +539,9 @@
 &phy3a {
 	status = "okay";
 };
+
+&lpc0_kcs2 {
```
Comment (ojayanth) [inline arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts:543]: Can you cleanup this

**Review by ojayanth — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts (hunk 1 above)
Comment (aasboddu) [reply arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts:543]: facing issues with rebasing moved to new PR.

https://github.com/AMDESE/linux-aspeed/pull/28

**Review by supven01 — CHANGES_REQUESTED** [review]
JTAG changes have come in this pull request. [Some unintentional changes ...Please re submit]

**Comment by aasboddu** [comment]
facing issues with rebasing moved to new PR. Closing this one

https://github.com/AMDESE/linux-aspeed/pull/28

**Pull request closed without merging**